spinn_front_end_common/interface/buffer_management/buffer_manager.py

```diff
@@ -544,24 +542,22 @@ def __count_regions(self):
         return n_regions_to_read, recording_placements
 
     def get_placement_data(self):
-        timer = Timer()
-        with timer:
-            with self._thread_lock_buffer_out:
-                if self._java_caller is not None:
-                    self.__get_data_for_placements_using_java()
+        FecTimer.start_category(FecTimer.BUFFER)
+        with self._thread_lock_buffer_out:
+            if self._java_caller is not None:
+                self.__get_data_for_placements_using_java()
+            else:
+                n_regions, recording_placements = self.__count_regions()
+                progress = ProgressBar(
+                    n_regions,  "Extracting buffers from the last run")
+                if get_config_bool(
+                        "Machine", "enable_advanced_monitor_support"):
+                    self.__old_get_data_for_placements_with_monitors(
+                        recording_placements, progress)
                 else:
-                    n_regions, recording_placements = self.__count_regions()
-                    progress = ProgressBar(
-                        n_regions,  "Extracting buffers from the last run")
-                    if get_config_bool(
-                            "Machine", "enable_advanced_monitor_support"):
-                        self.__old_get_data_for_placements_with_monitors(
-                            recording_placements, progress)
-                    else:
-                        self.__old_get_data_for_placements(
-                            recording_placements, progress)
-        with ProvenanceWriter() as db:
-            db.insert_category_timing(BUFFER, timer.measured_interval, None)
+                    self.__old_get_data_for_placements(
+                        recording_placements, progress)
+        FecTimer.end_category(FecTimer.EXTRACTING)
```

Comment:
Should the start and end category match here?

Reply:
Yes start and end should match.

Note: In this case I removed the Category BUFFER/EXTRACTING so this is now part of the RUN_LOOP
Instead added a Work type  EXTRACT_DATA  and used that in the EnergyReport  (for now)